### PR TITLE
Graph edges

### PR DESCRIPTION
**Pull request by M-Ravenmoore**
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: javascript/graphs/get-edges.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// will be a function to find the edges
+// time boxed to hour got the whiteboard will revisit on weekend.
```
Comment (adrianhuebner) [inline javascript/graphs/get-edges.js:2]: When you have a chance to come back to this add code and write some tests, then I will regrade for you!
